src/core/model/table/typed_column_data.h

```diff
@@ -204,32 +209,38 @@ class TypedColumnDataFactory {
                 }
                 return is_simple_date;
             };
+    inline static std::function<bool(std::string const&)> const kDoubleCheck =
+            [](std::string const& val) {
+                bool is_double = false;
+                try {
+                    std::size_t pos = 0;
+                    std::stod(val, &pos);
+                    if (pos == val.size()) {
+                        is_double = true;
+                    }
+                } catch (...) {
+                }
+                return is_double;
+            };
```

Comment:
I understand that this is not your code, but since you're optimizing type deduction, we can make it better:
- try block with a catch-all and not handling the exception is generally not a good practice, looks very strange
 
 We can make it better by utilizing `std::from_chars`, which is a modern way of converting strings to numbers, plus it's exception-free.
 Smth like
 ```cpp
 [](std::string const& val) {
        double result;
        auto [p, ec] = std::from_chars(val.data(), val.data() + val.size(), result);
        return ec == std::errc{} && p == val.data() + val.size();
    };
```